node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -3,7 +3,7 @@
 
 <div id="topnavigation-container" role="navigation" aria-label="__MSG__TOPNAV_ARIA_LABEL__">
     <!-- ACCESSIBILITY HELPERS -->
-    <a id="topnavigation-skip" class="sr-only oae-skip-link hide" href="#topnavigation-target" accesskey="__MSG__SKIP_TOP_MENU_KEY__">__MSG__SKIP_TOP_MENU__</a>
+    <a id="topnavigation-skip" class="sr-only oae-skip-link hide" href="#oae-page-content">__MSG__SKIP_TO_CONTENT__</a>
```

Comment:
This link has the `hide` class applied to it, causing it to never receive focus. I'm not sure why it was ever added and if it ever worked.